app/models/item.rb

```diff
@@ -3,7 +3,15 @@ class Item < ActiveRecord::Base
 
   validates_presence_of :name, :price, :quantity, :shelf_life_days
 
-  def expires
-    Date.today + shelf_life_days.days
+  scope :valid_items, lambda {
+    where('shelf_life_days > (current_date - date(created_at))')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.